packages/sdk/src/swap/services/ApiService.ts

```diff
@@ -61,7 +61,9 @@ const getQuote: BackendQuery<
   const { brokerCommissionBps, ...returnedRequestData } = quoteRequest;
   const params: QuoteQueryParams = {
```

Comment:
the `quote` endpoint of the api requires a `srcChain` and a `destChain` now to prepare for arbitrum (there will be two `USDC` assets with arbitrum)

Reply:
these were already mandator for the `getQuote` method so its not a bc break in the sdk, but it is a change in the interface of the (internal) api

Reply:
implemented in https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/550/commits/54ae86c1f33bdce4510331094b80ae7f384c4a1e